Source/Core/Scripting/ScriptList.cpp

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Scripting/ScriptList.h"
+
+namespace Scripts
+{
+void StartPendingScripts()
+{
+  for (int i = 0; i < m_scripts.size(); i++)
+  {
+    if (m_scripts[i].enabled && !m_scripts[i].backend)
+      m_scripts[i].backend = new Scripting::ScriptingBackend(m_scripts[i].path);
+  }
+  scripts_started = true;
+}
+
+// called on game close
+// maintain enabled flag, but delete backends
+void StopAllScripts()
+{
+  for (int i = 0; i < m_scripts.size(); i++)
+  {
+    if (m_scripts[i].backend)
+    {
+      delete m_scripts[i].backend;
+      m_scripts[i].backend = nullptr;
+    }
+  }
+  scripts_started = false;
+}
+
+std::vector<Script> m_scripts;
+bool scripts_started;
```

Comment:
The variable is called `m_scripts` despite no longer being a member of any struct or class. Perhaps `s_scripts` or `g_scripts` is preferable here instead.

A similar prefix would be recommended on `scripts_started`.

Reply:
Went for g prefix. s_ indicates static which isn't really correct here.